Buildings/issue1137_rename_scripts_temp.txt

```diff
@@ -0,0 +1,136 @@
+
+
+for ff in `find . \( -name '*.mo' -or -name '*.mos' -or -name '*.py' -or -name '*.txt' -or -name '*.svg'  \)`; do
+    list=(\
+
+# Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits
+
+    # in Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.SetPoints.ZoneTemperatures
+    
+     occCooSet occTDisSet \
+     TCooSet TZonCooSet \
+     THeaSet TZonHeaSet \
+     TCooOnMax TZonCooSetMax \
+     TCooOnMin TZonCooSetMin \
+     THeaOnMax TZonHeaSetMax \
+     THeaOnMin TZonHeaSetMin \
+     TCooWinOpe TZonCooSetWinOpe \
+     THeaWinOpe TZonHeaSetWinOpe \     
+
+    # in Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Reheat.SetPoints.ActiveAirFlow
+
+     VCooMax VDisCooSetMax_flow \
+     VMin VDisSetMin_flow \
+     VHeaMax VDisHeaSetMax_flow \
+     VMinCon VDisConMin_flow \
+     AFlo ZonAre \
+     outAirPerAre VOut_perAre_flow \
+     outAirPerPer VOut_perPer_flow \
+     VOccMinAir occVDisMin_flow \
+     VActCooMax VActCooMax_flow \
+     VActCooMin VActCooMin_flow \
+     VActMin VActMin_flow \
+     VActHeaMin VActHeaMin_flow \
+     VActHeaMax VActHeaMax_flow \
+
+    # Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Reheat.SystemRequests
+  
+     TCooSet TZonCooSet \
+     VDis VDis_flow \
+     VDisSet VDisSet_flow \
+     TDis \
+     cooSetDif_1 dTZon_ZonCooSet_1 \ # these are very long
+     cooSetDif_2 dTZon_ZonCooSet_2 \ 
+     disAirSetDif_1 dTDis_DisSet_1 \
```

Comment:
Do we need two times `Dis` in `Dis_Dis`?

Reply:
It's simply a T difference between ```TDis``` and ```TDisSet```. Let's ignore that it has the indexes (_1, _2) for now. ```dTDisSet``` is shorter and appealing, but it indicates only one of the variables defining the dT. ```dTDis_Set``` may be a bit too cryptic. Let me know if you have a preference.

Reply:
If it is a control error, you could call it `errTDis` (= set point minus measured quantity)

Reply:
That fits well, thanks. Do you have any preferences on ```dTDis_ZonSet_Max```? It's the maximum offset.

Reply:
I would need to know more context but `dTDisZonSetMax` or `errTDis(Max?)` may be options